Library/Formula/mu.rb

```diff
@@ -43,6 +43,10 @@ def install
     # shipped by default with Mac OS X is too old.
     ENV["EMACS"] = "no" if build.without? "emacs"
 
+    # Make --related work on 10.9
+    # https://github.com/djcb/mu/issues/380
+    ENV.append 'CXXFLAGS', '-std=c++11'
```

Comment:
10.8 doesn't ship with a libc++. If this needs C++11 instead use `ENV.cxx11` and/or `needs :cxx11`. Thanks!